Source/CoreClasses/Game.swift

```diff
@@ -8,21 +8,23 @@
 import Foundation
 
 ///Represents a digital game.
-public class Game {
+public class Game: NSObject {
 
-  public var appID: String
-  public var name: String
-  public var price: Double = 0.00
+  @objc public var appID: String
+  @objc public var name: String
+  @objc public var price: Double = 0.00
 
-  public init(gameName name: String, gamePrice price: Double) {
+  @objc public init(gameName name: String, gamePrice price: Double) {
 
     self.name = name
     self.price = price
     appID = ""
```

Comment:
If we are defaulting appID to be "" regardless, we might as well set it directly on line 13 instead of having an extra line of code

Reply:
See below comment instead of this

Reply:
Whoops, didn't mean to resolve, but I have been wanting to do something where I can almost just say Game.default and returns a default game, Why? I really like the idea. What do you think?

Reply:
Sounds pretty doable with a static computed variable although I think your initialisers will need to be marked as required. I'm also not too sure you can use default since it's a keyword 😓 
Nice idea though